### ISSUE TITLE
Fitting properly with absolute_sigma

### ISSUE THREAD

**Issue by ysBach** [python4astronomers/python4astronomers#32]
In the [ curve fitting example](https://github.com/python4astronomers/python4astronomers/blob/master/examples/curve_fitting/scipy_curve_fit.py), sigma is given but ``absolute_sigma`` option is not specified (False by default). This sometimes is an unexpected behavior. (same goes true in Matlab).

I think it is better to specify this option for more accurately describe the fitting error analysis.

**Comment by taldcroft** [comment]
Good point.  Are you interested in submitting a pull request to fix this?
